README.md

```diff
@@ -36,6 +38,87 @@ after your tests have run:
 mv -v coverage/.resultset.json coverage/.resultset${BUILDKITE_PARALLEL_JOB}.json
 ```
 
+### Multiple test suites
+
+If your repository has more than one test suite you'll need to format each artifact (coverage file) before the result can be summed and reported to codeclimate.
+
+To do this you'll need to configure multiple steps persisting the formated codeclimate compatable configuration files as artifacts between steps.
+
+This is best shown via an example;
+
+```
+  - label: ":rails: Tests"
+    key: "rails-tests"
+    command: "rails db:test:prepare && rails test"
+    env:
+      RUBY_OPT: "-W:deprecated"
+    depends_on:
+      - "build-base"
+    plugins:
+      - docker-compose#v3.7.0:
+          run: base
+          config: .buildkite/docker-compose.yml
+          volumes:
+            - "./coverage:/app/coverage"
+    artifact_paths:
+      - "coverage/**/*"
+
+  - label: ":codeclimate: Format Rails Coverage"
+    key: "codeclimate-format-rails-report"
+    plugins:
+      - jobready/codeclimate-test-reporter#v2.2:
+          artifact: "coverage/coverage.json"
+          input_type: simplecov
+          prefix: /app
+          report: false
+          file_prefix: "codeclimate.rails"
+    depends_on:
+      - "rails-tests"
+    artifact_paths:
+      - "coverage/codeclimate.**"
+
+  - label: ":react: Tests"
+    key: "react-tests"
+    command: "yarn test"
+    depends_on:
+      - "build-base"
+    plugins:
+      - docker-compose#v3.7.0:
+          run: base
+          config: .buildkite/docker-compose.yml
+          volumes:
+            - "./coverage:/app/coverage"
+    artifact_paths:
+      - "coverage/**/*"
+
+  - label: ":codeclimate: Format React Coverage"
+    key: "codeclimate-format-react-report"
+    plugins:
+      - jobready/codeclimate-test-reporter#v2.2:
+          artifact: "coverage/lcov.info"
+          input_type: lcov
+          prefix: /app
+          report: false
+          file_prefix: "codeclimate.react"
+    depends_on:
+      - "react-tests"
+    artifact_paths:
+      - "coverage/codeclimate.**"
+
+  - label: ":codeclimate: Report coverage"
+    key: "codeclimate-coverage-report"
+    plugins:
+      - jobready/codeclimate-test-reporter#v2.2:
+          artifact: "coverage/codeclimate**"
+          file_prefix: "codeclimate.*"
```

Comment:
Do we need this var defined? I'm confused by the use of `codeclimate.*` when the default is `codeclimate`. It seems unnecessary

Reply:
You don't, I've removed it